components/post_view/post_attachment.jsx

```diff
@@ -79,6 +47,8 @@ export default class PostAttachment extends React.PureComponent {
     }
 
     getCollapsedTextHTML() {
+        // TODO: this breaks markdown formatting when it e.g. cuts a ``` block terminator
+        // Should be collapsed using another method.
```

Comment:
Do we have anything to do here?

Reply:
not for this PR, but I noticed that if it's collapsing a markdown text, it can break e.g. a code block by cutting the terminator line.
so ultimately this should be removed and replaced with CSS collapse...

BUT from what I know there is work to move the collapsing to "post" level(designs at ICU channel), so then this piece of code could be removed at all